nextcord/voicerec/shared.py

```diff
@@ -0,0 +1,46 @@
+# SPDX-License-Identifier: MIT
+
+from enum import Enum
+from os import makedirs
+from os.path import dirname
+from pathlib import Path
+
+
+class TempType(Enum):
+    Memory = 0
+    File = 1
+
+
+class Formats(Enum):
+    MP3 = 0
+    MP4 = 1
+    M4A = 2
+    MKA = 3
+    MKV = 4
+    OGG = 5
+    PCM = 6
+    WAV = 7
+
+
+format_exports = {
+    Formats.MP3: "export_with_ffmpeg",
+    Formats.MP4: "export_with_ffmpeg",
+    Formats.M4A: "export_with_ffmpeg",
+    Formats.MKA: "export_with_ffmpeg",
+    Formats.MKV: "export_with_ffmpeg",
+    Formats.OGG: "export_with_ffmpeg",
+    Formats.PCM: "export_as_PCM",
+    Formats.WAV: "export_as_WAV",
+}

```

Comment:
why not map directly to functions?

Reply:
> why not map directly to functions?

I don't exactly remember rn, I although know that I considered mapping directly to the functions, I must've had a reason.

Reply:
changed to address issue